nilearn/image/tests/conftest.py

```diff
@@ -49,7 +49,9 @@ def img_4d_mni_tr2(img_4d_mni):
     img = image.new_img_like(
         img_4d_mni, data=img_4d_mni.get_fdata(), copy_header=True
     )
-    # Change the TR
+    # Add fake description
     header = img.header.copy()
+    header["descrip"] = b"Fake description"
```

Comment:
Added this because the `test_mean_img_copied_header` was still passing with `copy_header=False`